components/calendar/test/calendar.test.js

```diff
@@ -0,0 +1,61 @@
+import '../calendar.js';
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+
+const normalFixture = html`<d2l-calendar selected-value="2015-09-02T12:00Z"></d2l-calendar>`;
+
+describe('d2l-calendar', () => {
+
+	it('passes all axe tests', async() => {
+		const calendar = await fixture(normalFixture);
+		await expect(calendar).to.be.accessible();
+	});
+
```

Comment:
I think there's a good opportunity here to write actual unit tests for all the utility methods that you have outside of the component class. e.g. `getDatesInMonthArray`, etc.

Reply:
I had looked into that but I think they would have to not be private in order for me to do that (though I could be missing/forgetting about a way to do this).

Reply:
Yeah you'd need to export them. Not sure if we care that folks could import and use them... seems like an OK trade-off when it enables better testing.

Reply:
ok cool, yeah seems like a good call to me then.